### DISCUSSION TITLE
Purpose is Everywhere (in theory) (aka Purpose BS)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#26] [General]
## **UPDATE: posted at https://rufuspollock.com/post/purpose-bullshit**

Purpose bullshit describes the growing tendency for businesses to claim they are purpose-driven and doing good in the world -- whilst still doing classic profit-driven capitalist stuff. This is misleading -- hence the term "bullshit".

And it devalues real purpose. Elon Musk is not saving the world. Nike (or Coke) is not setting you free. You aren't an eco-warrior because you take a reusable cup at Starbucks.

> [!note]
> Originally created in GitLab by @rufuspollock on Nov 20, 2019, 23:24

## UPDATE 2023-12-28 - Hypocrisy is the tribute vice pays to virtue

I originally start this thread to highlight what i saw as an increasing trend of "purpose-washing" or "purpose bullsh**": the situation where businesses talk about how purpose-driven and social impact focused they are whilst doing classic profit-driven capitalist stuff.

Originally I thought this was simply indicative of classic advertising disingenuity: wrapping an unattractive "money-focus" in the flag of purpose and social commitment. A subtler version of advertising's tendency to dishonesty in pursuit of a more attractive presentation -- I still remember from my childhood the oil company Esso having as its main [brand image a tiger roaming in the wilderness](https://www.youtube.com/watch?v=Bmm-waULrhA) [running through the wilderness](https://www.youtube.com/watch?v=qz4SZ3D_Vc4).

### Hypocrisy is the tribute vice pays to virtue

However, I'm increasingly of the view that this "purpose BS" tendency is actually indicative of a profound and important shift to a post-modern/green cultural paradigm. After all, **hypocrisy is the tribute vice pays to virtue**. Rather than focusing on the vice -- i.e. the disingenuous purpose-washing -- let's focus on the virtue: the desire to prioritize purpose and social impact. That's a profound change in the orientation of business.

And we can see this shift everywhere. Just the other day I was talking to an old colleague who ran a successful tech consultancy in Switzerland. He explained that after pioneering holacracy a few years ago, their recent focus is pioneering the "purpose" steward-owned company concept following https://purpose-economy.org/ (whose founders i happen to know well). For me this was a perfect demonstration that this purpose-oriented steward ownership is clearly mainstream on the leading edge -- similar to how ecological attitudes (and platitudes) are now mainstream in most leading businesses (think VC, think silicon valley tech) etc.

And this is **wonderful**. This is **good news**.

Aspiration is the first sign of change -- even if it is not yet lived.

The challenge faced for most of these companies at the present is they are trapped in contradiction: they aspire to postmodernist values whilst still operating inside a modernist capitalist system. The result is often hypocrisy. I imagine it as similar to late romans having to suddenly try to behave as good Christians whilst still having to still fight and conquer to survive. It led to the same kind of hypocrisy -- the kind of hypocrisy we see in decayed form in the Godfather where you attend mass but still have to execute your brother and lie about it.

---

## 2022-07-25

Sprint book by partners at Google Ventures. From very last pages of book:

> At GV, we invest in startups because **we want them to change the world for the better**. We want you to change the world, too.

Not because we want them to make money etc. Because we want them to change the world (and for the better!)

## 2019-09-28

https://www.nytimes.com/2019/09/28/business/wework-juul-ebay-ceo.html - excellent listing of purpose bullshit in a discussion of wework and others. Here were some excerpts:

> “At the end of the day, it’s still mostly about stock price if you’re a public company C.E.O.,” Mr. Dickerson said. “When the rubber meets the road and you’re sitting in the room with investors, they are looking at spreadsheets and asking you about what the numbers are going to look like.” [Dickerson is the ex-CEO of Etsy]

Adam Neumann at WeWork

> "We are a community company committed to maximum global impact. Our mission is to elevate the world’s consciousness." S1 filing. https://www.sec.gov/Archives/edgar/data/1533523/000119312519220499/d781982ds1.htm

David Wenig - eBay

> “make a bigger difference around the world.” [Ed: You are a 2nd hand consumer goods platform] https://www.linkedin.com/pulse/my-annual-letter-ebay-shareholders-devin-wenig/

Marc Benioff

> “We’re in a world where we need leaders to improve the state of the world, not just the state of the bottom line,” Marc Benioff, the co-chief executive of Salesforce, said in an interview. “Every C.E.O. has this on their mind right now.”
>
> ...
>
> “You have to deliver performance in your business,” Mr. Benioff said. “That’s table stakes.”

Peloton -- fitness equipment [sells $2000 fitness bikes and interactive platform]

> "On the most basic level, Peloton sells happiness. But of course, we do so much more."

> ".. We are an innovation company transforming the lives of people around the world ...” [what about all the people who can't afford or don't need a $2k bike?]

![peloton-mission-promo](https://github.com/life-itself/community/assets/180658/b8c2a53c-07c4-424b-9161-6ebb625fc5ac)

### Dropbox -- cloud storage ...

> unleash the world’s creative energy by designing a more enlightened way of working.

### Uber -- a glorified taxi-booking service ...

> ignite opportunity by setting the world in motion.

### Spotify -- song streaming!

> ... unlock the potential of human creativity.

### Facebook

> "make the world more open and connected"

A) The reality is this high-falutin record does not actually translate to reality ...

=> and this points to a deeper issue in [digital] capitalism ... => to win you need capital fast (because its a race) => you sell out to investors who care about returns ... => even if you *were* authentic about your purpose (dubious to start with) you are now accountable to people who don't care about it ...

> The online marketplace Etsy went public in 2015. A quirky company from the outset, Etsy was helmed by Chad Dickerson, a beloved leader who prioritized generous employee benefits and earned the company B-Corp status, signifying a high level of social and environmental responsibility.
> 
> “I thought a lot about what, in addition to building shareholder value, a company can contribute to the world,” Mr. Dickerson said.
> 
> He didn’t have long to ruminate on such matters. Once Etsy was public, its stock started to fall. Soon, private equity firms were circling and activist investors were agitating for change. The Etsy board, which had been supportive of Mr. Dickerson’s agenda until then, reversed course, and he was unceremoniously fired.
> 
> “It’s not Milton Friedman’s 1970s shareholder value world anymore,” Mr. Dickerson said. “Except when it is.”

Later ...
 
> In 2015, David Crane was chief executive of NRG, one of the country’s largest power producers — and one of its largest polluters. Mr. Crane believed that NRG had a moral and business imperative to transition to renewable sources of power generation.
> 
> His vision for a clean energy company made Mr. Crane the talk of the industry, and animated idealistic employees. But when Mr. Crane asked the board to endorse a plan for NRG to be carbon neutral by 2040, they balked. One board member took him aside and said, “Are you crazy? You can’t say that.”
> 
> That director was right. When it became clear to investors that Mr. Crane was serious about his plans, NRG stock started to fall, and Mr. Crane was out.
> 
> In the end, it didn’t matter that Mr. Crane believed he was on the right side of history, or that his staff was behind him. “The fact that employees liked it was overwhelmed by the fact that the board didn’t like it, and investors didn’t care,” he said. (Mr. Crane, though, may have simply been ahead of his time. Since he lost his job, the cost of renewable energy has continued to decline, and several large utilities have pledged to become carbon neutral.)

To conclude:

> “At the end of the day, it’s still mostly about stock price if you’re a public company C.E.O.,” Mr. Dickerson said. “When the rubber meets the road and you’re sitting in the room with investors, they are looking at spreadsheets and asking you about what the numbers are going to look like.”

The deeper problems of Purpose BS is that it:

* Crowds out / drowns out real purpose -- it becomes a form of "purpose-washing" (you can imagine an arms merchant talking about how they "Enable security and peace" etc)
* Trivialises real purpose -- making a washing machine is not transforming the world
* Indulges the consumerist fantasy that we're / i'm making a difference just through buying.
  * Which in turn substitutes for political action. => this is the big issue
    * It tends to focus on symbolic and relatively trivial issues. 
  * It therefore perpetuates a dysfunctional "capitalism"
* Behind it: if capitalism works why do you need to go on about your "CSR" -- just do your job ...
  * So behind all of this is an implicit discontent with capitalism
* Obscures the hard fact that really its about making lots money often from monopolies

## 2019-05-08 - https://www.collaborativefund.com/about/

Also in the last decade, “doing good” became a driving force for building successful, impactful businesses.

Once seen as sacrificial to growth and returns, pursuing a social mission now plays a big role when attracting both customers and employees. We went from labeling do-gooders as the kids who got beat up at school to celebrating Elon Musk with Tesla and SolarCity.

...

And we’re not alone. We increasingly see more investors joining us at the intersection of for-profit and for-good. Established partnerships like Sequoia Capital and Founders Fund have evolved to stress world-changing aspirations. New entrants like Obvious Ventures, Social Capital, Sherpa Capital, and others have joined the fun as well.

# More miscellaneous examples (from Gitlab)

## DCVC

https://www.dcvc.com

![image](https://user-images.githubusercontent.com/180658/128873484-094a7779-1704-4170-bf91-dde34a270211.png)

> DCVC backs entrepreneurs solving trillion-dollar problems to multiply the benefits of capitalism for everyone while reducing its costs.

You have got to love that "multiply the benefits of capitalism for everyone" 😉

## FORE partnership

https://forepartnership.com/about/

"Purpose driven real estate investment partnership"

What is "purpose driven" about it (and what *is* the purpose?)

![image](https://user-images.githubusercontent.com/180658/128873509-6e6586d7-fb7f-47fb-8098-ec0af1487e10.png)

[About___FORE_Partnership.pdf](/uploads/586fe55645ead5509bd8a16cb3f5b002/About___FORE_Partnership.pdf)

## Capricorn Investment (ebay cofounder Jeff Skoll family office)

https://capricornllc.com

Ask yourself: are the world's greatest problems actually solvable via (just) market forces? And if so, is a lack of capital a problem? Is this the false consciousness of the 1% (or rather 0.001%)? (To be clear, I applaud Jeff Skoll's work and his commitment to active philanthropy. It might just be clearer to say: we're making as much money here as possible, purpose-driven or not, so that we can support the great philanthropic work we do elsewhere).

![image](https://user-images.githubusercontent.com/180658/128873525-c99d8abd-7f5e-4c87-ae28-9c77e344c63e.png)

## 2 April 2020 - beondeck

+purposebs +technosolutionism

"unique solution to a massive world problem". Really? Is that what tech startups are doing?

https://twitter.com/b_nicks11/status/1242459642908704768

> 7/ It’s hard to find the right co-founder and first hires, have the mindspace to have conviction around their **unique solution to a massive world problem**, and have the connections to quality investors to make it a reality. On Deck's vision is to make that easier and better.

![image](https://user-images.githubusercontent.com/180658/128873570-d3701925-0579-4d87-8c4a-9f08fdfa7c90.png)

## 19 July 2019

Purpose Bullshit tendency in modern capitalism e.g. eutopia.vc

![image](https://github.com/life-itself/community/assets/180658/ae095205-f40d-477c-a542-3b11e0621802)

![Pasted image 20230928114146](https://github.com/life-itself/community/assets/180658/22e06c0f-684a-43be-8762-2a419267ad88)

![Pasted image 20230928114212](https://github.com/life-itself/community/assets/180658/34a9995e-88bc-411a-957c-180a172abf6b)

Earlier screenshots:

![Pasted image 20230928114158](https://github.com/life-itself/community/assets/180658/84841841-fdc3-4772-a0cf-792aa2d573a9)

**Comment by rufuspollock** [comment]
## LiTA.co

#eco #purpose #france

> Investments that matter
> 
> On LITA.co, choose the world you want your savings to support. Invest directly in companies that take a real part in social and ecological transition.

![image](https://github.com/life-itself/community/assets/180658/b5d25052-5f1c-40f8-af85-335250a176b0)

![image](https://github.com/life-itself/community/assets/180658/8bcf5f11-5f5e-48b5-aac9-c5c790ae6808)

**Comment by martinprogrammer** [comment]
@rufuspollock 

> However, I'm increasingly of the view that this "purpose BS" tendency is actually indicative of a profound and important shift to a post-modern/green cultural paradigm. After all, hypocrisy is the tribute vice pays to virtue. Rather than focusing on the vice -- i.e. the disingenuous purpose-washing -- let's focus on the virture: the desire to prioritize purpose and social impact. That's a profound change in the orientation of business.

I agree, if the goal is to transition into a weller and wiser world - we shouldn't be scrutinising the motivation too harshly.

It might seem to be ethically problematic joining in forces with the groups whose ideology you are actively trying to defeat. 
But is that not the "wiser" component?

1) Understand different motivations
2) Communicate shared vision taking also into account opposing motivations
3) Build alliances and relationships
4) Provide incentives

I welcome inevitable criticism for sounding Machiavellian, but pragmatism is missing in an idealist perspective and that hurts us.

**Reply by rufuspollock** [reply]
I didn't mean joining forces with those you are trying to defeat here ... more to see the "good intentions underlying the hypocrisy/BS".
